Google Play Music/ColorWheel.cs

```diff
@@ -518,6 +519,51 @@ public static Color HslToRgb(HslColor hsl)
             return Color.FromArgb(rgb[0], rgb[1], rgb[2]);
         }
 
+
+        /// <summary>
+        /// Convert RGB Color to a hex value
+        /// </summary>
+        /// <param name="rgb">RGB color</param>
+        /// <returns>Hex value of RGB color</returns>
+
+        public static string RgbToHex(Color rgb)
+        {
+            string hex = "#" + rgb.R.ToString("X2") + rgb.G.ToString("X2") + rgb.B.ToString("X2");
+            return hex.ToUpper();
+        }
+        
+
+        /// <summary>
+        /// Convert hex value to RGB color
+        /// </summary>
+        /// <param name="hex">Hex value of color</param>
+        /// <returns>Color</returns>
+        public static Color HexToRgb(string hex)
+        {
+            Color rgb = Color.FromArgb(HexToInt(hex));
+            return rgb;
+        }
+
+        /// <summary>
+        /// Convert hex value to int value. Filters leading # character.
+        /// </summary>
+        /// <param name="hex">Hex value of color</param>
+        /// <returns>int value of color, or black if invalid</returns>
+
+        private static int HexToInt(string hex)
+        {
+
+            int rgb = 0;
```

Comment:
Can the default RGB be configured to be the same as the default hexdecimal code in the default user settings file (found here)  

https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/Google%20Play%20Music/App.config#L28